requirements/core-requirements.txt

```diff
@@ -19,3 +19,4 @@ markdown<4,>=3.3
 Jinja2<4,>=2.11; platform_system != 'Windows'
 Jinja2<4,>=3.0; platform_system == 'Windows'
 matplotlib<4
+Flask-SQLAlchemy>=3.0.0,<=3.0.3
```

Comment:
```suggestion
```

Can we remove this because the auth feature is optional?

Reply:
sure! I rewrote it to use just sqlalchemy instead, similar to model registry & tracking store